deployment/gunicorn/commcarehq_wsgi.py

```diff
@@ -0,0 +1,11 @@
+import os
+from manage import _set_source_root_parent, _set_source_root
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+
+_set_source_root_parent('submodules')
+_set_source_root(os.path.join('corehq', 'ex-submodules'))
+_set_source_root(os.path.join('custom', '_legacy'))
```

Comment:
might be nice to pull this out into a shared function in case we add paths in the future